modules/document_repository/jsx/uploadForm.js

```diff
@@ -164,46 +169,80 @@ class DocUploadForm extends Component {
    *********************************************************************************/
 
   /**
-   * Upload file
+   * Upload file(s)
    */
-  uploadFile() {
+  uploadFiles() {
     // Set form data and upload the media file
-    let formData = this.state.formData;
-    let formObject = new FormData();
-    for (let key in formData) {
-      if (formData[key] !== '') {
-        formObject.append(key, formData[key]);
+    try {
+      this.setState({uploadInProgress: true});
+      let formData = this.state.formData;
+      let formObject = new FormData();
+      for (let key in formData) {
+        if (formData[key] !== '') {
+          if (key === 'files' &&
+            document.querySelector('.fileUpload').multiple) {
+              Array.from(formData[key]).forEach((file) => {
+                formObject.append('files[]', file);
+              });
+          } else {
+            formObject.append(key, formData[key]);
+          }
+        }
       }
-    }
 
-    fetch(this.props.action, {
-      method: 'POST',
-      cache: 'no-cache',
-      credentials: 'same-origin',
-      body: formObject,
-    })
-    .then((resp) => {
-      console.error(resp);
-      if (resp.ok) {
-        swal.fire('Upload Successful!', '', 'success').then((result) => {
-          if (result.value) {
-            this.setState({formData: {}});
-            this.props.refreshPage();
+      fetch(this.props.action, {
+        method: 'POST',
+        cache: 'no-cache',
+        credentials: 'same-origin',
+        body: formObject,
+      })
+        .then((resp) => {
+          console.error(resp);
```

Comment:
why is resp always unconditionally logged as an error?

Reply:
I'm not sure why that was done but it was not introduced by this PR. It seems to be a forgotten debugging snippet.

Reply:
I agree. Done.